### PR TITLE
Fixed splash page

### PR DESCRIPTION
**Pull request by kobe1104**
Fixed h1 from disappearing and overlapping with form. Fixed nickname font size so it's center in the text box at any size.

### CODE REVIEW

File: assets/css/main.css [hunk 1]
```diff
@@ -22,14 +20,24 @@
   align-items: center;
 }
 
+.splash-inner-container {
+  display:flex;
+  position: relative;
+  width: 960px;
+  height:960px;
```
Comment (deeptailor) [inline assets/css/main.css:27]: Why did you choose 960 x 960 px ?

File: assets/css/main.css [hunk 2]
```diff
@@ -63,7 +72,8 @@
   background: white;
   border: 1px solid gray;
   border-radius: 5px;
-  font-size: 20px;
+  /*font-size: 20px;*/
+  font-size: 1.2vw;
```
Comment (deeptailor) [inline assets/css/main.css:76]: why are you setting the font size relative to view width?

**Review by deeptailor — COMMENTED** [review]
Check feedback on css files

**Comment by kobe1104** [comment]
That's actually random, but it seems to work so I didn't change it.

**Pull request merged**